WelcomeHome/WelcomeHome.Services/DTO/EventDTO.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WelcomeHome.Services.DTO
+{
+    internal class EventDTO
+    {
+        public DateTime? Date { get; set; }
+
+        [Required]
```

Comment:
For this we will write validatiors (FluentValidator). Because these conditions are not checked anywhere